docs/deltaWithMSTestV1.md

```diff
@@ -1,12 +1,27 @@
 ## Behaviour changes w.r.t MSTest V1
- 
+
 Listed are the difference in behaviour of MSTest V2 w.r.t MSTest V1 :
-1. Test methods from base classes defined in another assembly are now discovered and run from the derived Test class. This brings in a consistent behavior with derived test class types. If this behavior is not required for compat reasons it can be changed back using the following runsettings:
+1.	MSTest V2 sets the direction for how we intend to evolve the MSTest framework [[more...]](https://blogs.msdn.microsoft.com/devops/2016/06/17/taking-the-mstest-framework-forward-with-mstest-v2/).
+2.	MSTest V2 is "open source" [[more...]](https://blogs.msdn.microsoft.com/devops/2017/04/05/mstest-v2-is-open-source/).
+3.	Uniform app-platform support – this is a converged implementation that offers uniform app-platform support across .NET Framework, .NET Core and ASP.NET Core, and UWP [[more...]](https://blogs.msdn.microsoft.com/devops/2016/09/01/announcing-mstest-v2-framework-support-for-net-core-1-0-rtm/).
+4.	The implementation is fully cross platform (Windows, Linux, Mac) [[more...]](https://blogs.msdn.microsoft.com/devops/2017/04/05/mstest-v2-is-open-source/).
+5. Provides a uniform, single end-user extensibility mechanism. [[more...]](https://blogs.msdn.microsoft.com/devops/2017/07/18/extending-mstest-v2/).
+6. Delivered as a NuGet package [[more...]](https://www.nuget.org/packages/MSTest.TestFramework/).
+7.	Provides a uniform DataRow support, for all MSTest based test projects [[more...]](https://blogs.msdn.microsoft.com/devops/2017/02/25/mstest-v2-now-and-ahead/).
+8.	Provides the ability to place the TestCategory attribute at the level of a class or assembly [[more...]](https://blogs.msdn.microsoft.com/devops/2017/02/25/mstest-v2-now-and-ahead/).
+9. Test methods from base classes defined in another assembly are now discovered and run from the derived Test class. This brings in a consistent behavior with derived test class types. If this behavior is not required for compat reasons it can be changed back using the following runsettings:
  ```
  <RunSettings>    
   <MSTest> 
     <EnableBaseClassTestMethodsFromOtherAssemblies>false</EnableBaseClassTestMethodsFromOtherAssemblies> 
   </MSTest> 
 </RunSettings>
 ```
-2. The TestCleanup method on a TestClass is invoked even if its corresponding TestInitialize method fails. See [#250](https://github.com/Microsoft/testfx/issues/250) for details.
+10. The TestCleanup method on a TestClass is invoked even if its corresponding TestInitialize method fails. See [#250](https://github.com/Microsoft/testfx/issues/250) for details.
+
+Here are the features that are not supported:
+1.	Tests cannot be included into an "Ordered Test".
+2.	The adapter does not support being configured via a .testsettings file.
+3.	The adapter does not support test lists specified as a .vsmdi file.
+4.	CodedUITest/Web and Load Test are not supported.
+5.	Association with a testcase item in TFS is not supported.
```

Comment:
Should we also add here that MSTestV2 is supported from net45 and beyond and is not supported for lower .net frameworks?

Reply:
Done. I added that as point no. 5 in the MSTest V2 in the preceding section.

Reply:
👍